### PR TITLE
Fix imports from packit/ogr

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Stop importing packit.github_service.

### CODE REVIEW

File: packit_service/service/events.py [hunk 1]
```diff
@@ -58,9 +57,31 @@ class WhitelistStatus(enum.Enum):
 class Event:
     def __init__(self, trigger: JobTriggerType):
         self.trigger: JobTriggerType = trigger
+        self._user_config = None
+
+    @property
+    def user_config(self):
```
Comment (phracek) [inline packit_service/service/events.py]:
```
def user_config(self) -> Config:
```

Comment (rpitonak) [inline packit_service/service/events.py]: user_config is going to be replaced by service_config in #60

Comment (lachmanfrantisek) [reply packit_service/service/events.py]: Do I need to fix that here?

Comment (rpitonak) [reply packit_service/service/events.py]: no need, I will do while rebasing my PR.

**Pull request merged**